user-impl/src/main/java/com/example/auction/user/impl/UserServiceImpl.java

```diff
@@ -41,7 +41,8 @@ public UserServiceImpl(PersistentEntityRegistry registry, ActorSystem system, Ma
     public ServiceCall<UserRegistration, User> createUser() {
         return user -> {
             UUID uuid = UUID.randomUUID();
-            PUser pUser = new PUser(uuid, user.getName(), user.getEmail());
+            String password = PUserCommand.hashPassword(user.getPassword());
+            PUser pUser = new PUser(uuid, user.getName(), user.getEmail(),password);
             return entityRef(uuid)
                     .ask(new PUserCommand.CreatePUser(pUser))
                     .thenApply(done -> Mappers.toApi(pUser));
```

Comment:
If the `CreatePUser` constructor is auto-generated by Lombok, then you don't need to create the `pUser` here. Instead, you can use the reply value from the `CreatePUserCommand`:

```java
return entityRef(uuid)
  .ask(new PUserCommand.CreatePUser(user.getName(), user.getEmail(), password))
  .thenApply(createdUser -> Mappers.toApi(createdUser));
```